build/minio.sh

```diff
@@ -72,7 +81,11 @@ EOM
 case "${1}" in
         # Alphabetically sorted
         install_minio)
-            time -p install_minio
+            if [[ ${#@} -eq 2 ]]; then
+                time -p install_minio ${2}
+            else
+                time -p install_minio
+            fi
```

Comment:
Instead of calling `install_minio` twice here. Let's just call install_minio from here will all the passed arguments and then handle things in the definition of install_minio.

Reply:
fixed